src/generator.rs

```diff
@@ -96,6 +99,29 @@ struct Generator<'a> {
 }
 
 impl Generator<'_> {
+    fn generate_api_resources_options(self, api: Api) -> anyhow::Result<()> {
+        self.generate_api_resources_options_inner(api.resources.values())
+    }
+    fn generate_api_resources_options_inner<'a>(
```

Comment:
Please let's have an empty line between `fn` definitions, consistently. (same below)

```suggestion
    }

    fn generate_api_resources_options_inner<'a>(
```

Reply:
fixed